chatsky/slots/slots.py

```diff
@@ -117,29 +116,8 @@ class ExtractedValueSlot(ExtractedSlot):
     """Value extracted from :py:class:`~.ValueSlot`."""
 
     is_slot_extracted: bool
-    extracted_value: Any
-    default_value: Any = None
-
-    @field_serializer("extracted_value", "default_value", when_used="json")
-    def pickle_serialize_values(self, value):
-        """
-        Cast values to string via pickle.
-        Allows storing arbitrary data in these fields when using context storages.
-        """
-        if value is not None:
-            return pickle_serializer(value)
-        return value
-
-    @field_validator("extracted_value", "default_value", mode="before")
-    @classmethod
-    def pickle_validate_values(cls, value):
-        """
-        Restore values after being processed with
-        :py:meth:`pickle_serialize_values`.
-        """
-        if value is not None:
-            return pickle_validator(value)
-        return value
+    extracted_value: Union[BaseModel, JsonValue]
```

Comment:
Slots store exceptions on failure (which are not serializable).
I think we should store exception representation instead.